src/components/package-show.js

```diff
@@ -22,100 +20,96 @@ export default function PackageShow({ vendorPackage, packageTitles }, { intl })
   };
 
   return (
-    <div data-test-eholdings-package-details>
-      <Paneset>
-        <Pane defaultWidth="100%">
-          {vendorPackage.isResolved ? (
-            <div>
-              <div style={{ margin: '2rem 0' }}>
-                <KeyValueLabel label="Package">
-                  <h1 data-test-eholdings-package-details-name>
-                    {packageRecord.packageName}
-                  </h1>
-                </KeyValueLabel>
+    <div style={{ padding: '0 1rem' }} data-test-eholdings-package-details>
```

Comment:
also, i know i've seen `em` used in other parts of the application.  is it kosher to mix-and-match `rem` and `em`, or is it better to uniformly use one of them?  i had to google them to understand what they actually meant, but if i'm understanding it correctly it sounds like we should consistently use one or the other

Reply:
In my opinion, generally use `rem`, but `em` can be helpful when basing proportions on the current text size. Like padding for a button.